src/main/kotlin/no/nav/syfo/Environment.kt

```diff
@@ -50,29 +55,82 @@ fun getDbConfig(): DbEnvironment {
     )
 }
 
-private fun localEnvironment(): Environment {
-    return objectMapper.readValue(File(localEnvironmentPropertiesPath), Environment::class.java)
+fun jobEnvironment(): JobEnvironment {
+    val (
+        remote,
+        _,
+        _,
+        kafkaBootstrapServersUrl,
+        kafkaSchemaRegistryUrl,
+        stsUrl,
+        _,
+        pdlUrl,
+        dkifUrl,
+        serviceuserUsername,
+        serviceuserPassword,
+        _,
+        _,
+        _,
+        _,
+        _,
+        dbEnvironment
+    ) = if (isLocal()) localEnvironment() else remoteEnvironment()
+    return JobEnvironment(
+        remote,
+        kafkaBootstrapServersUrl,
+        kafkaSchemaRegistryUrl,
+        stsUrl,
+        pdlUrl,
+        dkifUrl,
+        serviceuserUsername,
+        serviceuserPassword,
+        dbEnvironment
+    )
 }
 
 data class Environment(
-    val remote: Boolean,
+    override val remote: Boolean,
     val applicationPort: Int,
     val applicationThreads: Int,
-    val kafkaBootstrapServersUrl: String,
-    val stsUrl: String,
+    override val kafkaBootstrapServersUrl: String,
+    override val kafkaSchemaRegistryUrl: String,
+    override val stsUrl: String,
     val syfosyketilfelleUrl: String,
-    val pdlUrl: String,
-    val dkifUrl: String,
-    val serviceuserUsername: String,
-    val serviceuserPassword: String,
+    override val pdlUrl: String,
+    override val dkifUrl: String,
+    override val serviceuserUsername: String,
+    override val serviceuserPassword: String,
     val syfosmregisterUrl: String,
     val syfosmregisterScope: String,
     val aadAccessTokenUrl: String,
     val clientId: String,
     val clientSecret: String,
-    val dbEnvironment: DbEnvironment
-)
+    override val dbEnvironment: DbEnvironment
+): CommonEnv
+
+data class JobEnvironment(
+    override val remote: Boolean,
+    override val kafkaBootstrapServersUrl: String,
+    override val kafkaSchemaRegistryUrl: String,
+    override val stsUrl: String,
+    override val pdlUrl: String,
+    override val dkifUrl: String,
+    override val serviceuserUsername: String,
+    override val serviceuserPassword: String,
+    override val dbEnvironment: DbEnvironment
+): CommonEnv
 
+interface CommonEnv {
```

Comment:
Kanskje bedre med `CommonEnvironment` for å være konsistent ( det finnes feks `DbEnvironment`)?